src/options.rs

```diff
@@ -42,6 +42,9 @@ pub struct Options {
     /// Ignore a specified column.
     #[structopt(short = "I", number_of_values = 1)]
     pub ignore: Vec<String>,
+
+    #[structopt(short = "i", number_of_values = 1)]
```

Comment:
Let's leave only the long form (`--derive`) here for now

Reply:
Done.